reV/supply_curve/point_summary.py

```diff
@@ -113,55 +113,6 @@ def __init__(self, gid, excl, gen, tm_dset, gen_index, excl_dict=None,
 
         self._apply_exclusions()
 
-    def _apply_exclusions(self):
```

Comment:
Just to confirm, this method and _resource_exclusion() were not modified, just moved?

Reply:
yeah just re-organized a bit